scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/attestation/DummyAttestationService.scala

```diff
@@ -46,15 +46,10 @@ class DummyAttestationService extends AttestationService {
             }
             val nonce: String = "test"
             val response: JSONObject = new JSONObject()
-            val number = new BigInteger(quote)
-            val zero = new BigInteger("0")
-            var verifyQuoteResult = true
+            val verifyQuoteResult = Random.nextBoolean()
```

Comment:
Random true of false is not good design. I think we can get "true" from quote string, then return true. Otherwise return false

Reply:
update in https://github.com/intel-analytics/BigDL/pull/4594/files#diff-83601ee9b506b76581cdfd8d7811c08da5e7b0fde78efb91870e57e1fe4e5cb5R52